src/content/vintage/platform-overview/security/operational-layers/index.md

```diff
@@ -31,38 +31,37 @@ owner:
   - https://github.com/orgs/giantswarm/teams/team-planeteers
 ---
 
-A Giant Swarm installation has several operational layers. At Giant Swarm we use the term operational layers in order to indicate the different layers of code that may require access by different users for different activities. Operational layers in this context are in effect a representation of a separation of concerns both on an operational and on a security level. In this document we will define the layers and explain the operational model.
+A Giant Swarm installation has several operational layers. At Giant Swarm we use the term operational layers in order to indicate the different layers of the system that may require access by different users for different activities. Operational layers in this context are in effect a representation of a separation of concerns both on an operational and on a security level. In this document we will define the layers and explain the operational model.
 
 ## Operational layers
 
 We will go through the operational layers one by one from the bottom (infrastructure) to the top (user space) and explain the intended operational model by defining (typical) users and permission levels. The layers are:
 
 1. [Infrastructure](#infrastructure)
-2. [Giant Swarm management cluster](#management-cluster)
-3. [Giant Swarm REST API](#giant-swarm-api)
-4. [User Space](#userspace)
+2. [Giant Swarm management cluster and Platform API](#management-cluster)
+3. [User Space](#userspace)
 
 ### Infrastructure {#infrastructure}
 
-The infrastructure layer covers the area on top of actual (or virtual) machines, networking, etc., which is managed by Giant Swarm SREs (Site Reliability Engineers). It is usually accessed through VPN and bastion hosts as well as through the cloud provider APIs if applicable.
+The infrastructure layer covers the area on top of actual (or virtual) machines, networking, etc., which is managed by Giant Swarm SREs (Site Reliability Engineers). It is usually accessed through VPN and bastion hosts as well as through the cloud provider APIs if applicable. In most installations there's also ssh access through [Teleport]({{< relref "/vintage/platform-overview/security/cluster-security/cluster-access/#admin-access-via-teleport" >}}).
```

Comment:
where to move cluster access, do we have a new place already, I found Teleport in vintage weird

Reply:
yeah we did not move that one yet. Maybe as I child of architecture page too

Reply:
should we move that in a separate PR? I think I would not want to extend this one for it